test/system/710-kube.bats

```diff
@@ -106,24 +106,23 @@ metadata.labels.app        | =  | ${pname}
 metadata.name              | =  | ${pname}
 
 spec.hostname                              | =  | $pname
-spec.restartPolicy                         | =  | Never
```

Comment:
Any particular reason not to change this one from `Never` to `null`, like the others? Why are you removing this check entirely?

Reply:
I can open another PR to test that out, but would like to get this one in today before we need to cut the release.